docs/hub-deployment-guide/runbooks/phase3/initial-hub-setup.md

```diff
@@ -0,0 +1,176 @@
+     touch ./config/clusters/$CLUSTER_NAME/common.values.yaml
+     ```
+
+4. **Run the deployer to generate a sample basic hub configuration**
+
+   The easiest way to add new configuration is to use the deployer to generate an initial sample config.
+
+   You will be asked to input all the information needed for the command to run successfully. Follow the instructions on the screen and using the information provided to you, fill in all the fields.
+
+   - **Run the deployer command below to generate config for the specific hub configuration:**
+     ```bash
+     deployer generate hub-asset main-values-file
+     ```
+
+   - **Run the deployer commands below to generate config for the common hubs configuration:**
+     ```bash
+     deployer generate hub-asset common-values-file
+     ```
+
+   ```{tip}
+   Each `*.values.yaml` file is a Zero to JupyterHub configuration, and you can customize whatever you like.
+
+   You can also look at the entries for similar hubs under the same cluster folder, copy / paste one of them, and make modifications as needed for this specific hub.
+   For example, see the hubs configuration in [the 2i2c Google Cloud cluster configuration directory](https://github.com/2i2c-org/infrastructure/tree/HEAD/config/clusters/2i2c).
+   ```
+
+4. **Setup the relevant Authentication Provider with relevant credentials**
+
+   See [](enable-auth-provider) for steps on how to achieve this.
+
+5. **Then reference these files in a new entry under the `hubs` key in the cluster's `cluster.yaml` file**
+
+   You can use the `deployer generate hub-asset` subcommand to generate the relevant entry to insert into cluster.yaml file.
+
+   ```bash
+   deployer generate hub-asset cluster-entry --cluster-name $CLUSTER_NAME --hub-name $HUB_NAME --hub-type $HELM_CHART_TYPE
+   ```
+
+   ```{warning}
+   Please pay attention to all the fields that have been auto-generated for you by this command and change every one that doesn't match the community's requirements or was not rendered correctly before copying-pasting it into the relevant files.
+   ```
+
+6. **Add the new cluster to CI/CD**
+
+   ```{important}
+   This step is only applicable if the hub is being deployed to a new cluster that has been recently created by following the previous runbooks.
+   ```
+
+   To ensure the new cluster and its hubs are appropriately handled by our CI/CD system, please add it as an entry in the following places:
+
+      - The [`deploy-hubs.yaml`](https://github.com/2i2c-org/infrastructure/blob/008ae2c1deb3f5b97d0c334ed124fa090df1f0c6/.github/workflows/deploy-hubs.yaml#L121) GitHub workflow has a job named [`upgrade-support-and-staging`](https://github.com/2i2c-org/infrastructure/blob/18f5a4f8f39ed98c2f5c99091ae9f19a1075c988/.github/workflows/deploy-hubs.yaml#L128-L166) that needs to list of clusters being automatically deployed by our CI/CD system. Add an entry for the new cluster here.
+
+6. **Create a Pull Request with the new hub entry**
```

Comment:
Line numbers repeat `6`. I think if you repeat `1`, it will when rendered show up as 1,2,3,4,5 etc, so an option is also to just do `1.` over and over.

I appreciate the only-use-1-strategy as it makes it unreasonable to reference the specific numbers, which is something I find good to avoid as it creates fragile / hard-to-maintain docs. It also avoids having `10.` that gets indented further than `9.` etc.